src/components/FormPills/FormPills.test.ts

```diff
@@ -21,6 +21,18 @@ describe('template', () => {
         expect(wrapper.vm.removeItem).toBeCalledWith(0);
     });
 
+    // For some reason the trigger('blur') doesn't work. Anyone got ideas what
+    // might be off?
```

Comment:
I had faced before, If I'm not mistaken some devs said that double focus works, somehow ignores first focus